test/test_helper.rb

```diff
@@ -30,20 +30,24 @@
 
 module Coverband
   module Test
+    TEST_DB = 2
+
+    def self.redis
```

Comment:
@danmayer - added a test redis instance that all of our tests can use. This way we all our in the same db and our database is being cleaned out properly.